src/main/java/com/iexec/core/task/TaskRepository.java

```diff
@@ -44,6 +44,42 @@
     @Query("{ 'currentStatus': {$in: ?0} }")
     List<Task> findByCurrentStatus(List<TaskStatus> statuses, Sort sort);
 
+    @Query("{ 'currentStatus': {$in: ?0} , 'tag': {$ne: ?1} }")
```

Comment:
I think there is no need for this as spring will parse the current name of the method and reconstruct the request.

Reply:
You're right. Let's remove it!